GDJS/Runtime/runtimescene.ts

```diff
@@ -1158,7 +1187,7 @@ namespace gdjs {
      * in exceptional use cases where you need to loop through all objects,
      * and it won't be performant.
      *
-     * @returns The list of all runtime objects on the scnee
+     * @returns The list of all runtime objects on the scene
```

Comment:
This typo won't exist soon.

Reply:
Alright, I'll remove it to avoid conflicts